.changeset/proud-bulldogs-share.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/sidebar': minor
```

Comment:
```suggestion
'@twilio-paste/sidebar': major
```

Reply:
Keeping it a minor until we get all the pieces in it.